src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/active-alert-list/active-alert-list.component.ts

```diff
@@ -96,6 +92,25 @@ export class ActiveAlertListComponent extends PrometheusListHelper implements On
         prop: 'startsAt',
         cellTransformation: CellTemplate.timeAgo,
         flexGrow: 1
+      }
+    ];
+    this.columns = [
+      {
+        name: $localize`Name`,
+        prop: 'labels.alertname',
+        cellClass: 'fw-bold',
+        flexGrow: 2
+      },
+      {
+        name: $localize`Summary`,
+        prop: 'annotations.summary',
+        flexGrow: 3
+      },
+      ...this.innerColumns.slice(1),
+      {
+        name: $localize`Count`,
```

Comment:
These are alert instances that are created. 
```suggestion
        name: $localize`Instance`,
```

Reply:
<img width="4578" height="734" alt="image" src="https://github.com/user-attachments/assets/593ddef3-f48b-40bb-a5ec-1db0f3bfb96d" />